src/components/buttons.js

```diff
@@ -0,0 +1,16 @@
+import PropTypes from 'prop-types';
+
+function Button({ name }) {
+  return (
+    <>
+      <button type="button">{name}</button>
+    </>
```

Comment:
- [ ]  You need t0 use class-based components as per the project's requirement. The objective of this milestone is to practice React class components. Kindly refactor this component to a class-based component. :pray: